### ISSUE TITLE
In ST on Windows, clicking on a tab sometimes opens that file in a new window

### ISSUE THREAD

**Issue by tssajo** [jisaacks/GitGutter#228]
This is a very-very annoying bug. It is described here:
https://sublimetext.com/forum/viewtopic.php?f=3&t=11303

It was tracked by someone to the sidebar panel. Finally the GitGutter plugin was identified as being the culprit, see here:
http://www.sublimetext.com/forum/viewtopic.php?f=3&t=10289&start=10#p59021

I can also independently verify that this problem started happening on my PC (I have 64bit Windows 7 with latest ST3 installed) right after I installed GitGutter. In my config file "`live_mode`" was set to `false` and I still encountered the problem.

It would be nice if someone could track down and fix this long outstanding issue for us Windows users.

**Comment by tssajo** [comment]
### Update:
The problem can be elminitad by adding
```json
"focus_change_mode": false
```
into:

*(Preferences > Package Settings > GitGutter > Settings - User)*

Obviously, changes won't be evaluated when a view gets the focus... :(

**Comment by rchl** [comment]
From what I can understand from linked forum thread, the problem boils down to system triggering a drag of the tab when sublime's UI gets blocked during tab switch. If that is the case then #321 should help here.
If you still can reproduce this problem then please check once again when new version gets released (>1.3.0).

**Comment by tssajo** [comment]
@rchl I did not experience this problem for years. In fact, I even forgot that I filed this report... It was an accurate bug report back then, I remember now for sure. But once I added this to GitGutter's config file:
```
"focus_change_mode": false
```
the problem went away and never surfaced again. So I consider this to be the permanent solution as it solved this problem for one and a half years for me. This is definitely good enough for me.
